### DISCUSSION TITLE
About Bootstrap 5 Css

### DISCUSSION THREAD

**Discussion by ratneshupadhyay-kombee** [Power-Components/livewire-powergrid#1552] [BUG]
I have setup a powergrid but its not apply bootstrap5 css even i put link from the bootstrap site in app.blade file direct then its working why this happen  

       <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
        <link rel="stylesheet" href="{{asset('assets/livewire-powergrid/bootstrap5.css')}}">
        
        I try both way show 
        This link Work 
          <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
          but by default  livewire-powergrid/bootstrap5.css not working

**Comment by abhaypithadiya** [comment]
Hey! Did you configure the theme in config/livewire-powergrid.php?
https://livewire-powergrid.com/get-started/powergrid-configuration.html#_2-choose-a-css-theme

**Reply by ratneshupadhyay-kombee** [reply]
Yes , But Its not working, I setup all things one by one

**Reply by abhaypithadiya** [reply]
Are you getting any errors in the console?

**Reply by ratneshupadhyay-kombee** [reply]
No , But Design is not getting on my table , is short css is not apply

**Reply by abhaypithadiya** [reply]
Can you try ``` php artisan optimize:clear``` and check it once?

**Reply by ratneshupadhyay-kombee** [reply]
I will try but still same issue

**Reply by abhaypithadiya** [reply]
Can you remove the link attribute from the head and add the below in resources/js/app.js?
```
// resources/js/app.js

import './../../vendor/power-components/livewire-powergrid/dist/bootstrap5.css'
```

**Reply by ratneshupadhyay-kombee** [reply]
i use this also but not work
